lib/image/icc_profile.js

```diff
@@ -0,0 +1,187 @@
+import zlib from 'zlib';
+class ICCProfile {
+  static extractFromJPEG(jpeg) {
+    let pos = 2;
+    const buffers = [];
+    while (pos < jpeg.length) {
```

Comment:
Testing this gives me an error, because `const marker = jpeg.readUInt16BE(pos);` https://www.geeksforgeeks.org/node-js-buffer-readuint16be-method/.

If I change this to while (pos < jpeg.length - 2) {, it finishes sucessfully.